test/bindings.js

```diff
@@ -51,7 +51,7 @@ const readyData = Buffer.from('READY');
 
 // Test our mock binding and the binding for the platform we're running on
 bindingsToTest.forEach((bindingName) => {
-  const Binding = require(`../lib/bindings/${bindingName}`);
+  const Binding = require(`../dist/bindings/${bindingName}`);
```

Comment:
don't know maybe done that in a hurry your right the tests probally don't need to use dist :+1: